src/Prefixer.js

```diff
@@ -34,7 +34,12 @@ export function prefix (value, length) {
 			return WEBKIT + value + replace(value, /(\w+).+(:[^]+)/, WEBKIT + 'box-$1$2' + MS + 'flex-$1$2') + value
 		// align-self
 		case 5443:
-			return WEBKIT + value + MS + 'flex-item-' + replace(value, /flex-|-self/, '') + value
+			// center, end, start, stretch
+			return (
+				~indexof(['stretch', 'center', 'end', 'start'], substr(value, length + 1, sizeof(value) - 1))
```

Comment:
TODO: convert to regex, not Array.indexOf.

Reply:
Fixed in https://github.com/thysultan/stylis/pull/285/commits/817ff44b0bbc87f950e25b9e35f72d7531f2ee81